### PR TITLE
Make sure you can use the bundle without having twig installed

### PR DESCRIPTION
**Pull request by Nyholm**
Currently the bundle actually has a hard dependency on Twig. 
With this PR you may opt-out of using Twig.

### CODE REVIEW

**Comment by Nyholm** [comment]
Hm, Im not sure if this really works. I still get: 

> Symfony\Component\Debug\Exception\ClassNotFoundException: Uncaught exception 'Symfony\Component\Debug\Exception\ClassNotFoundException' with message 'Attempted to load class "Twig_Extension" from the global namespace.Did you forget a "use" statement?' in /home/sites/foo/bar/vendor/ekino/newrelic-bundle/Twig/NewRelicExtension.php:20

File: DependencyInjection/EkinoNewRelicExtension.php [hunk 1]
```diff
@@ -108,5 +108,9 @@ public function load(array $configs, ContainerBuilder $container)
             ->replaceArgument(4, $transaction_naming_service)
             ->replaceArgument(5, $config['using_symfony_cache'])
         ;
+
+        if (!$config['twig_enabled']) {
```
Comment (jderusse) [inline DependencyInjection/EkinoNewRelicExtension.php]: What's about checking if the class `Twig_Environment` exists?

Comment (Tobion) [reply DependencyInjection/EkinoNewRelicExtension.php]: Agree, the flex way is to enable things when they are installed.

Comment (Nyholm) [reply DependencyInjection/EkinoNewRelicExtension.php]: Sure, will do

**Comment by Nyholm** [comment]
Finally I found this PR again and completed it. 

I also noticed that @ruudk did #128 which is a duplicate of this PR.

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -29,6 +29,7 @@ public function getConfigTreeBuilder()
             ->fixXmlConfig('deployment_name')
             ->children()
                 ->booleanNode('enabled')->defaultTrue()->end()
+                ->booleanNode('twig')->defaultTrue()->end()
```
Comment (Tobion) [inline DependencyInjection/Configuration.php]: just set the default to `class_exists(\Twig_Environment::class)` or maybe we on't need a config at all

Comment (Nyholm) [reply DependencyInjection/Configuration.php]: I may want to turn this off even though I have Twig installed.

**Comment by Nyholm** [comment]
@Tobion Are you okey with the current state of this PR?

**Comment by Tobion** [comment]
As I proposed in https://github.com/ekino/EkinoNewRelicBundle/pull/126/files#r173443429 the default should be changed.

File: DependencyInjection/Configuration.php (hunk 2 above)
Comment (ruudk) [reply DependencyInjection/Configuration.php]: So something like this:
```php
->booleanNode('twig')->defaultValue(class_exists(\Twig_Environment::class))->end()
```

**Comment by Nyholm** [comment]
I've updated the PR and rebased on master. 

I just got write permissions to this repo but I do not want to merge my own PRs without approval. Could you do a final review?

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**